### ISSUE TITLE
keeps reloading until blocked

### ISSUE THREAD

**Issue by dchmelik** [mozilla/contain-facebook#947]
- Facebook Container Version: 2.3.11
- Operating System + Version: Slackware64 15+current
- Firefox Version: 115.0.3ESR
- Other installed Add-ons + Version + Enabled/Disabled-Status:
Name 	Type 	Version 	Enabled
British English Dictionary (Darmeth)	dictionary	3.1.3	true
British English Dictionary (Marco Pinto)	dictionary	3.2.2	true
English United States Dictionary	dictionary	91.0	true
'Improve YouTube!' 🎧 (for YouTube & Videos)	extension	4.20	true
AAAA: The Anti-Atob-Anti-Adblocker	extension	2	true
Absolute Enable Right Click & Copy	extension	1.3.8	true
AdBlocker for YouTube™	extension	0.3.7	true
Add Search	extension	1.1.1	true
Add-on List-o-matic 9000	extension	1.5	true
Add-ons Search Detection	extension	2.0.0	true
Amazon.com	extension	1.6	true
Anti-Adblock Blocker	extension	0.0.7	true
Autoplay Settings for YouTube	extension	2.0.1	true
Bing	extension	1.5	true
Block Site	extension	0.4.9	true
breadcrumbus	extension	1.0.5	true
Broken Link Checker	extension	0.3.1	true
Bulk Media Downloader	extension	0.3.1	true
CheckThemAll	extension	1.5	true
Chrome Store Foxified	extension	3.4	true
Consent Blocker	extension	0.3.8	true
Cookie Quick Manager	extension	0.5rc2	true
Copy PlainText	extension	1.13	true
Custom Scrollbars	extension	4.2.2	true
Decentraleyes	extension	2.0.17	true
Dictionary Anywhere	extension	1.1.0	true
Disconnect	extension	20.3.1.1	true
Dissenter Extension	extension	0.1.10	true
Don't Accept image/webp	extension	0.8.2	true
Don't track me Google	extension	4.28	true
Download All Images	extension	0.8.0	true
DownThemAll!	extension	4.10.1	true
DuckDuckGo	extension	1.4	true
Easy Video Downloader	extension	0.2.0	true
Easy Youtube Video Downloader Express	extension	18.4	true
eBay	extension	1.4	true
Enhancer for YouTube™	extension	2.0.119.1	true
EPUBReader	extension	2.0.13	true
Error 404 Wayback Machine	extension	1.2	true
F.B Purity - Cleans up Facebook	extension	36.8.0.0	true
Facebook Container	extension	2.3.11	true	@contain-facebook
Flagfox	extension	6.1.64	true
Flash Player 2022	extension	0.5.1	true
Forecastfox (fix version)	extension	4.26	true
Go Up	extension	1.1.0	true
Google	extension	1.4	true
Google Search "View Image" Button	extension	1.5	true
Google search link fix	extension	1.6.10	true
Grammar Checker & Paraphraser – LanguageTool	extension	7.1.13	true
Greasemonkey	extension	4.11	true
History in Threads	extension	2.3.4	true
HTML Validator	extension	0.9.9.5	true
I don't care about cookies	extension	3.4.7	true
Library Extension	extension	2023.717.1	true
Link Analyzer	extension	2.0.2	true
Link Gopher	extension	2.4.4	true
Link Status Redux	extension	3.20	true
Linkificator	extension	3.3.3	true
Livemarks	extension	3.5	true
mail.com MailCheck	extension	6.4.0	true
Multiple Tab Handler	extension	3.1.11	true
Myibidder Auction Bid Sniper for eBay	extension	2.0.1	true
MySessions	extension	2021.6.0	true
NextPlease	extension	0.10.0	true
NoScript	extension	11.4.26	true
Nuke Anything	extension	2.4	true
Open Image in Old Tab	extension	0.1.3	true
OverbiteNX	extension	0.2	true
Ping Blocker	extension	1.0.0	true
Polisis	extension	2.0	true
Popup To Tab	extension	0.2.2	true
Print Edit WE	extension	29.5	true
Privacy Badger	extension	2023.6.23	true
Privacy Pass	extension	3.0.5	true
Privacy Possum	extension	2019.7.18	true
Redirect AMP to HTML	extension	2.1.0	true
Search by Image	extension	5.7.0	true
Simple Translate	extension	2.8.1	true
Snip it! button for eBay	extension	2.4.11	true
SoundFixer	extension	1.3.0	true
SponsorBlock for YouTube - Skip Sponsorships	extension	5.4.12	true
Stylebot	extension	3.1.3	true
Stylus	extension	1.5.33	true
Swift Selection Search	extension	3.48.0	true
Tab Mix - Links	extension	0.1.0	true
Tab Mix - Rename Tab	extension	1.0.2	true
Tab Session Manager	extension	6.12.1	true
TinEye Reverse Image Search	extension	2.0.4	true
To Google Translate	extension	4.2.0	true
uAutoPagerize	extension	0.1.9.3	true
uBlock Origin	extension	1.51.0	true
User-Agent Switcher and Manager	extension	0.5.0	true
Video DownloadHelper	extension	8.0.0.0	true
View Source	extension	61.3	true
Weather	extension	5.0.9	true
Web Archives	extension	4.1.0	true
Web to PDF	extension	1.3	true
Webmail Ad Blocker	extension	5.14	true
Wikipedia (en)	extension	1.3	true
Windscribe - Free Proxy and Ad Blocker	extension	3.4.8	true
X-notifier (for Gmail™,Hotmail,Yahoo,AOL...)	extension	4.2.23	true
YesScript2	extension	4.3	true
Yet Another Smooth Scrolling WE	extension	2.0.10	true
YouTube NonStop	extension	0.9.1	true
Youtube-shorts block	extension	1.4.1	true
Zoom Page WE	extension	19.13	true
Language: English (GB)	locale	115.0.20230710.165010	true
Language: English (US)	locale	115.0.20230710.165010	true
some disabled ones

### Actual behavior
Keeps reloading facebook.com until it says I'm blocked for doing things too fast/much.  Won't let me open the site in a plain personal container or without one.

### Expected behavior
Don't reload unless one presses reload.  Don't force usage.

### Steps to reproduce
1. Open facebook.com (when logged-in).

**Comment by JAMonschke** [comment]
I am seeing this behavior as well with various websites, that goes away when I disable the "Facebook" addon. Additionally, it cycles so quickly that I have found it impossible to close the cycling tab without resorting to restarting Firefox. I have uninstalled "Facebook container" for the moment, and deleted the Facebook entry in "Multi-account containers". I will try reinstalling "Facebook container"  the next time that I restart Firefox and see if that helps.
